SYCL/ESIMD/api/functional/ctors/ctor_default.cpp

```diff
@@ -0,0 +1,158 @@
+//==------- ctor_default.cpp  - DPC++ ESIMD on-device test ----------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu || level_zero
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// Test for esimd default constructor.
+
+#include "../common.hpp"
+
+using namespace cl::sycl;
+using namespace sycl::ext::intel::experimental::esimd;
+using namespace esimd_test::api::functional;
+
+// Dummy kernel for submitting some code into device side.
+template <typename DataTypeT, int TestNumber, int NumElems>
+struct kernel_for_simd;
+
+// Dummy kernel for submitting some code into device side.
+template <typename DataT, int NumElems, typename T> struct Kernel;
+
+// Struct that provide method that let get current test case description and
+// method that let call simd constructor for initializer test case
```

Comment:
provide -> provides
let get current -> lets getting the current
let call -> lets calling
```suggestion
// This struct provides a method returning the test case description and a worker method
// testing the default simd constructor used in RHS of assignment.
```
Please consider fixing descriptions of similar structs  below.

Reply:
May be mentioning a logic of implementation is not a better Idea... So, comments for these structures were reworked
they can be found [here](https://github.com/intel/llvm-test-suite/pull/565/commits/d56ee8f4b7fd7c3b8f8d3015a45caef1e6bda23e#diff-6d44d6893e30c56cec1c2e698ea43849081d8b092f7a87532bd567294d14ed69L28-L67)